lib/resque/worker.rb

```diff
@@ -491,20 +491,18 @@ def self.all_heartbeats
     # Returns a list of workers that have sent a heartbeat in the past, but which
     # already expired (does NOT include workers that have never sent a heartbeat at all).
     def self.all_workers_with_expired_heartbeats
-      workers = Worker.all
       heartbeats = Worker.all_heartbeats
       now = data_store.server_time
 
-      workers.select do |worker|
```

Comment:
Could we indicate why we are using `all_heartbeats` instead of `all` here and link to this PR in a code comment?

This code is very confusing and it will be helpful to a future reader to give more context/details about what we are doing here.

Reply:
I left a comment.

https://github.com/resque/resque/pull/1751/files#diff-a43b4a5dd5107cefd407ce6da9c354cbf3c466f1e422df6f96efb003ac84b9b7R494